src/classes.py

```diff
@@ -2,11 +2,20 @@
 # constructor
 
 # YOUR CODE HERE
+class LatLon():
+    def __init__(self, lat, lon):
+        self.lat = lat
+        self.lon = lon
         
 # Make a class Waypoint that can be passed parameters `name`, `lat`, and `lon` to the
 # constructor. It should inherit from LatLon. Look up the `super` method.
 
 # YOUR CODE HERE
+class Waypoint(LatLon):
+    def __int__(self, name, lat, lon):
+        self.name = name
+        self.lat = lat
+        self.lon = lon
```

Comment:
I would like to see you add methods so you can properly print out the objects. you also need to instantiate the classes to complete the assignment. You'll get a chance to work with __repr__ and __str__ in the adventure game so it's not a big deal if you don't come back and finish this. But if you have any issues getting started with those methods just hmu and I'll be more than happy to help. This is a critical bit of information however so I want to make sure you have a handle on this before the sprint.